metadata.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+# return version
```

Comment:
It returns not just version but service metadata. Update comment.

Reply:
Not yet done.

Reply:
Done